docs/languages/vscode-microprofile.md

```diff
@@ -4,4 +4,15 @@ sidebar_position: 6
 
 # VS Code Microprofile
 
-[Visit the GitHub](https://github.com/redhat-developer/vscode-microprofile)
\ No newline at end of file
+[![Link to GitHub](https://img.shields.io/badge/GitHub-redhat%E2%80%93developer%2Fvscode%E2%80%93microprofile-GitHub?style=for-the-badge&logo=github&logoColor=white&color=black)](https://github.com/redhat-developer/vscode-microprofile)
+[![Visual Studio Marketplace](https://img.shields.io/visual-studio-marketplace/v/redhat.vscode-microprofile?style=for-the-badge&label=VS%20Marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile)
+[![Installs](https://img.shields.io/visual-studio-marketplace/i/redhat.vscode-microprofile?style=for-the-badge)](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile)
+
+vscode-microprofile provides IDE features that help write MicroProfile applications.
```

Comment:
I'd suggest changing vscode-microprofile to either VS Code Microprofile or Tools for Microprofile as it's listed on the marketplace. I think the lowercase beginning the sentence looks a little odd.

Other than that looks great!

Reply:
Alright, I changed it to Tools for MicroProfile